Signal/src/ViewControllers/ColorPickerViewController.swift

```diff
@@ -429,7 +429,7 @@ private class MockConversationViewItem: NSObject, ConversationViewItem {
         return 2
     }
 
-    func firstValidGalleryAttachment() -> TSAttachmentStream? {
+    func firstValidAlbumAttachment() -> TSAttachmentStream? {
```

Comment:
There's a bunch of places I missed during the "gallery" -> "album" rename.  I've fixed them here.